### PR TITLE
Welcome bot for contributors

### PR DESCRIPTION
**Pull request by VenkyOS**
Signed-off-by: Venkataraman Padmanabhan <venkataraman.padmanabhan@seagate.com>

A GitHub bot that welcomes new users based off maintainer defined comments that must be located in .github/config.yml

It combines 3 plugins which has already being tested in CORTX repo,

- new-issue-welcome

- new-pr-welcome

- first-pr-merge

### CODE REVIEW

**Review by knekrasov — APPROVED** [review]

**Comment by suvratjoshi** [comment]
As stale bot file is maintained as `stale.yml`, so this bot's config file name can be changed to `welcome_bot.yml` if this is expected to be welcome-bot only configuration. If its expected to have other bots' configs as well, in case any added in future, then this file's name can be changed to `bots_config.yml` or something, for better understanding.
I think having `bot` in the filename gives more clarification.

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Configuration for new-issue-welcome
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening this issue. A contributor should be by to give feedback soon. In the meantime, please check out the [contributing guidelines](https://github.com/Seagate/cortx/blob/main/CONTRIBUTING.md) and explore other [ways you can get involved](https://github.com/Seagate/cortx/).
```
Comment (suvratjoshi) [inline .github/config.yml:4]: Looks like, this sentence needs correction
`A contributor should be by to give feedback soon.`

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+# Configuration for new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Thanks for your contribution in opening this pull request! Now you can be rewarded with a CORTX sticker by requesting [cortx sticker](https://www.seagate.com/promos/cortx-stickers)
+
```
Comment (suvratjoshi) [inline .github/config.yml:10]: This sentence can be added added here I think, similar to above one : 
`A reviewer should review your PR soon.'

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Comment by mukul-seagate11** [comment]
@all, please review & merge the PR

**Comment by knekrasov** [comment]
@mukul-seagate11 The patch is reviewed already - I can see unresolved discussions.
@VenkyOS  can you please respond?

**Pull request closed without merging**